### ISSUE TITLE
VimuxPromptCommand does not open a new split if there is an ongoing process in a different split

### ISSUE THREAD

**Issue by benediktms** [preservim/vimux#199]
The `VimuxPromptCommand` does not create a new horizontal split below vim, if there is already a split open that has an ongoing process. Probably the best example of this would be starting a some sort of development server and having vim open in the second pane.

**Comment by alerque** [comment]
Are you talking about some split you opened manually or a previous one created by Vimux automatically?

**Comment by benediktms** [comment]
@alerque This is a split I opened manually

**Comment by alerque** [comment]
Okay I see what you mean. My usage is usually either to manage splits myself *or* let this do it for me in a dedicated window, but not mix and match like that.

I'd be happy to facilitate some kind of fix for this is somebody comes up with a PR, but it might be a while before I get around to it myself.

There is already a mechanism for using a specific pane name, what this probably entails is setting up the default to something the user won't have running already and checking for it even when a custom name is not specified. This may or make not make sense as the default but at least as a configuration option I'd be fine with including it to cover this usage scenario.

Care to take a stab at it yourself?

**Comment by benediktms** [comment]
@alerque I would be to be quite honest my knowledge with vimscript is extremely superficial (basically just enough to be able to manage my configs) :(

**Comment by alerque** [comment]
Fair enough. I guess we can hope somebody else sees this and feel like diving in...
